hooks/check

```diff
@@ -11,19 +11,34 @@ fi
 
 ok=yes
 vault="secret/$GENESIS_VAULT_PREFIX"
-domain="$(lookup params.static_ip)"
+static_ip="$(lookup params.static_ip)"
+domain="$(lookup params.external_domain "$static_ip")"
 for cert in nginx/ssl_certificate; do
   if ! safe exists "$vault/$cert"; then
     describe "  #R{✘}  $vault/$cert [#Y{MISSING}]"
     ok=no
   else
-    if safe --quiet x509 validate "$vault/$cert" --for "$(lookup params.static_ip)" >/dev/null 2>&1; then
-      describe "  #G{✔}  $vault/$cert [#G{OK}]"
+    if [[ $domain == $static_ip ]]; then
```

Comment:
Quote the variables

Reply:
You shouldn't need to quote inside of the double brackets; Bash handles the "one-ness" of the values for you (it's the point of `[[` vs `[`)